src/neuroconv/tools/nwb_helpers/_configuration_models/_base_dataset_io.py

```diff
@@ -204,12 +171,18 @@ def __str__(self) -> str:
 
         return string
 
-    @root_validator
+    @model_validator(mode="before")
```

Comment:
`model_validator(mode="before")` is equivalent to the old `root_validator` in that it checks the key/value pairs passed to `__init__`, but technically prior to actually calling the `__init__` on those values

Reply:
Thanks for writting an explanation.